merkledag/merkledag.go

```diff
@@ -371,27 +410,27 @@ func EnumerateChildrenAsync(ctx context.Context, ds DAGService, root *Node, set
 func fetchNodes(ctx context.Context, ds DAGService, in <-chan []key.Key, out chan<- *Node, errs chan<- error) {
 	defer close(out)
```

Comment:
but, in the case of an error, i'm not sure if its imperative we process every node received, or if we error out asap